Programs/doremon.py

```diff
@@ -297,7 +297,7 @@ def Doraemon():
     black_aankha()
 
 
-if __name__ == '__main__':
+if __name__ == '__main__git status':
```

Comment:
@bhaveshgarg2000 this doesn't seem right to me.

Reply:
Let me check onces & update on it